packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,58 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import Logger from 'ember-metal/logger';
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor(assert) {
+    super();
+
+    this.assert = QUnit.assert;
```

Comment:
@chancancode - ah yes, that makes more sense. Let me make those changes real quick.